src/Components/Publishing/Sections/ImageCollection.tsx

```diff
@@ -48,11 +48,25 @@ function renderImages(images, dimensions, gutter, sectionLayout, width) {
     let renderedImage
     if (image.type === "image") {
       renderedImage = (
-        <Image image={image} sectionLayout={sectionLayout} width={imageSize.width} height={imageSize.height} />
+        <Image
+          images={images}
```

Comment:
Instead of passing a callback through context we're passing the `images` and `slideIndex` property from the ImageCollection components through props.